.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "girder"
+  "extends": "girder"
```

Comment:
You just had to ask. :grimacing: 

So actually, I've been working on adding an EditorConfig file, to programmatically define standards on this. The bottom line is that a number of other the `.json` files in Girder are using 2 spaces:
* https://github.com/girder/girder/blob/master/package.json
* All those in https://github.com/girder/girder/tree/master/plugins/item_tasks/plugin_test

The main exception seems to be older test artifacts and `plugin.json` files. The test artifacts aren't developed on, and plugin definitions seem to be moving towards mostly YAML.

Thus, I picked 2 spaces as the standard for JSON. I agree that 4 would be more consistent with our JS and Python files (though not CMake, Pug, or Stylus).

I really don't care as long as it ends up consistant, and I'd be just as happy to go with 4, as JSON is super-easy to automatically reformat via tools.